assets/docs/jeongukjae/models/distilbert_multi_cased_L-6_H-768_A-12/1.md

```diff
@@ -0,0 +1,87 @@
+# Module jeongukjae/distilbert_multi_cased_L-6_H-768_A-12/1
+
+A small, fast, cheap and light Transformer model trained by distilling multilingual BERT base cased model.
+
+<!-- asset-path: https://storage.googleapis.com/jeongukjae-tf-models/distilbert/distilbert-base-multilingual-cased.tar.gz -->
+<!-- network-architecture: transformer -->
+<!-- task: text-embedding -->
+<!-- license: apache-2.0 -->
+<!-- fine-tunable: true -->
+<!-- format: saved_model_2 -->
```

Comment:
Would you mind adding some language tags here so it's easier to find for users? Considering https://github.com/google-research/bert/blob/master/multilingual.md#list-of-languages, we already support a few languages:

```bash
<!-- language: ar -->
<!-- language: bn -->
<!-- language: bg -->
<!-- language: ca -->
<!-- language: zh-cn -->
<!-- language: zh-tw -->
<!-- language: da -->
<!-- language: en -->
<!-- language: et -->
<!-- language: fi -->
<!-- language: fr -->
<!-- language: de -->
<!-- language: el -->
<!-- language: he -->
<!-- language: hi -->
<!-- language: id -->
<!-- language: it -->
<!-- language: ja -->
<!-- language: ko -->
<!-- language: nl -->
<!-- language: no -->
<!-- language: pl -->
<!-- language: pt -->
<!-- language: ro -->
<!-- language: ru -->
<!-- language: es -->
<!-- language: sv -->
<!-- language: ta -->
<!-- language: tr -->
<!-- language: uk -->
<!-- language: ur -->
<!-- language: vi -->
```